backend/app/member_handlers.go

```diff
@@ -0,0 +1,102 @@
+package app
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/codescalersinternships/Flyspray/models"
+	"github.com/gin-gonic/gin"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+	"gorm.io/gorm"
+)
+
+type createMemberInput struct {
+	ProjectID int  `json:"project_id" validate:"required"`
+	Admin     bool `json:"admin_bool"`
+}
+
+func (a *App) createNewMember(c *gin.Context) (interface{}, Response) {
+	memberInput := createMemberInput{}
+	if err := c.ShouldBindJSON(&memberInput); err != nil {
+		log.Error().Err(err).Send()
+		return nil, BadRequest(errors.New("error binding json data"))
+	}
+	userID, exists := c.Get("user_id")
```

Comment:
there is a misunderstanding here. 
the user ID is the one who can add (create) a new member for the project, hence it's not the ID of the member.
you should check if the user ID has an admin role for the project that allows him to add a new member
the member ID should be given through your input
same for updateOwnership func